content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -238,92 +249,62 @@ $ kubectl get no ${NODE_NAME} -o json | jq '.status.config'
-remove the `Node.Spec.ConfigSource` field.
+To reset the Node to use the configuration it was provisioned with, edit the
+Node using `kubectl edit node ${NODE_NAME}` and remove the
+`Node.Spec.ConfigSource` field.
 
-### Observe that the Node is using its local default configuration
+#### Observe that the Node is using its local default configuration
 
-After removing this subfield, you should eventually observe that `Node.Status.Config`
-has become empty, as all config sources have been reset to `nil` (indicating the local
+After removing this subfield, `Node.Status.Config` eventually becomes
+empty, since all config sources have been reset to `nil`, which indicates that the local
 default config is `assigned`, `active`, and `lastKnownGood`), and no error is reported.
 
 {{% /capture %}}
 
 {{% capture discussion %}}
 ## Kubectl Patch Example
-As mentioned above, there are many ways to change a Node's configSource.
-Here is an example command that uses `kubectl patch`:
 
-```
+You can change a Node's configSource using several different mechanisms.
+This example uses `kubectl patch`:
+
+```bash
 kubectl patch node ${NODE_NAME} -p "{\"spec\":{\"configSource\":{\"configMap\":{\"name\":\"${CONFIG_MAP_NAME}\",\"namespace\":\"kube-system\",\"kubeletConfigKey\":\"kubelet\"}}}}"
 ```
 
 ## Understanding how the Kubelet checkpoints config
 
 When a new config is assigned to the Node, the Kubelet downloads and unpacks the
-config payload as a set of files on local disk. The Kubelet also records metadata
+config payload as a set of files on the local disk. The Kubelet also records metadata
 that locally tracks the assigned and last-known-good config sources, so that the
 Kubelet knows which config to use across restarts, even if the API server becomes 
 unavailable. After checkpointing a config and the relevant metadata, the Kubelet 
-will exit if the assigned config has changed. When the Kubelet is restarted by the
-babysitter process, it will read the new metadata, and use the new config.
+exits if it detects that the assigned config has changed. When the Kubelet is
+restarted by the OS-level service manager (such as `systemd`), it reads the new
+metadata and uses the new config.
 
 The recorded metadata is fully resolved, meaning that it contains all necessary
 information to choose a specific config version - typically a `UID` and `ResourceVersion`.
 This is in contrast to `Node.Spec.ConfigSource`, where the intended config is declared
 via the idempotent `namespace/name` that identifies the target ConfigMap; the Kubelet 
 tries to use the latest version of this ConfigMap.
 
-It can sometimes be useful to inspect the Kubelet's config metadata and checkpoints
-when debugging a Node. The structure of the Kubelet's checkpointing directory is as follows:
+When you are debugging problems on a node, you can inspect the Kubelet's config
+metadata and checkpoints. The structure of the Kubelet's checkpointing directory is:
 
-```
+```none
```

Comment:
Come to think of it, would it be better style if I reformatted this as a nested unordered-list?

Reply:
I think it's better as is. We use this formulation elsewhere in the docs. The `none` is just to prevent syntax highlighting.